<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>week</class>
 <widget class="QWidget" name="week">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>442</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>696</width>
    <height>442</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>696</width>
    <height>442</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="ChangeFormat">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Смена формата</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Setting">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>410</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="day_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>81</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="date_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>date</string>
    </property>
   </widget>
   <widget class="QLabel" name="frame_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="day_deg_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>day_deg_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="night_deg_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>night_deg_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="weather_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>weather_1</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="City">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>City</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="day_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>81</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="date_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>date</string>
    </property>
   </widget>
   <widget class="QLabel" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="day_deg_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>day_deg_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="night_deg_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>night_deg_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="weather_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>weather_1</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="day_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>81</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="date_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>date</string>
    </property>
   </widget>
   <widget class="QLabel" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="day_deg_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>day_deg_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="night_deg_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>night_deg_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="weather_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>weather_1</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="day_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>81</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="date_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>date</string>
    </property>
   </widget>
   <widget class="QLabel" name="frame_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="day_deg_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>day_deg_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="night_deg_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>night_deg_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="weather_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>weather_1</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="day_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>81</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="date_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>date</string>
    </property>
   </widget>
   <widget class="QLabel" name="frame_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="day_deg_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>day_deg_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="night_deg_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>night_deg_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="weather_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>weather_1</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="day_6">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>90</y>
     <width>81</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="date_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>date</string>
    </property>
   </widget>
   <widget class="QLabel" name="frame_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="day_deg_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>day_deg_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="night_deg_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>night_deg_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="weather_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>weather_1</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="day_7">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>90</y>
     <width>81</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="date_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>date</string>
    </property>
   </widget>
   <widget class="QLabel" name="frame_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="day_deg_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>day_deg_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="night_deg_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>night_deg_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="weather_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>weather_1</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
